test/WebJobs.Script.Tests.Integration/PreJIT/JitPreparesTest.cs

```diff
@@ -33,5 +33,19 @@ public void ColdStart_JitFailuresTest(string fileName, double threshold)
             // using 1% as approximate number of allowed failures before we need to regenrate a new PGO file.
             Assert.True(failurePercentage < threshold, $"Number of failed PGOs are more than {threshold} percent! Current number of failures are {failedPrepares}. This will definitely impact cold start! Time to regenrate PGOs and update the {fileName} file!");
         }
+
+        [Theory]
+        [InlineData(WarmUpConstants.JitTraceFileName)]
+        [InlineData(WarmUpConstants.LinuxJitTraceFileName)]
```

Comment:
I think we should have 2 different tests here, one for Windows and one for Linux and each with different line count.

Reply:
Got it. What should the line count be for Windows?

Reply:
I think, for windows, we should just use the existing assert (% of failures)
just for linux, instead of % of failures use line count (until we can figure out something better)
Btw these tests will be failing locally as well and if required we should feel free to adjust the threshold of 6500 to something else as required.

Reply:
Yes, leaving % of failures for windows and moving to checking file size for linux is OK!

Reply:
sounds good to me :)